WatchParty/client/src/components/MoviePage/MoviePage.js

```diff
@@ -0,0 +1,94 @@
+import React from 'react';
+import { Button } from 'react-bootstrap'
+
+import theMovieDb from 'themoviedb-javascript-library'
+
+import './MoviePage.css'
+
+theMovieDb.common.api_key = 'bd5800f4f98c685b042cd33a1a790365'
+theMovieDb.common.base_uri = 'https://api.themoviedb.org/3/'
+
+class MoviePage extends React.Component{
+
+    constructor(props){
+        super(props)
+        
+        this.state={
+            movieItem: [],
+            trailerKey: ""
+        }
+
+        this.fetchMovieData = this.fetchMovieData.bind(this);
+        this.defaultPoster = this.defaultPoster.bind(this);
+    }
+
+    defaultPoster(event){
+        event.target.src = "/default.png"
+    }
+
+    fetchMovieData(){
+        theMovieDb.movies.getById(
+            {"id": this.props.match.params.movieId }, 
+            data => {
+                var response = JSON.parse(data)
+                this.setState({movieItem: response,})
+            }, 
+            err => console.log(err)
+        );
+
+        theMovieDb.movies.getVideos(
+            {"id": this.props.match.params.movieId },
+            data => {
+                var response = JSON.parse(data)
+                if(response.results.length > 0){
+                    this.setState({trailerKey: response.results[0].key})
+                    console.log(this.state.trailerKey) 
+                }  
+            },
+            err => console.log(err)
+        );
+    }
+
+    componentDidMount(){
+        this.fetchMovieData();
+    }
+
+    render () {
+        return (
+            <div className="info-container">
+              <div className="poster-container">
+                <img onError={this.defaultPoster} src={`https://image.tmdb.org/t/p/w1280${this.state.movieItem.poster_path}`} width="260px" height="380px" alt="movie poster"></img>
```

Comment:
If we use anything like a url path like `https://image.tmdb.org/t/p/w1280 `, is it worth it to move this to a constants file and reference it as a var from there?

Reply:
If we want to, we already have dotenv. Though we may want to upload a file to the drive with all the url we're replacing with 'Process.env'.